### ISSUE TITLE
Use Properties in Custom searches

### ISSUE THREAD

**Issue by ryanjamurphy** [tadashi-aikawa/obsidian-another-quick-switcher#205]
Thanks for this plugin!

It'd be wonderful to scope a custom search by property. E.g., `[type:project]` would search only notes that have a `project` value for the `type` property.

Similarly, `[status:open]` would search only notes with an `open` `status`.

Obviously things can get more adventurous: scoping search by date range for date-type properties, or searching a range of possible values, etc. — all of these would be quite exciting!

**Comment by MitchWagnerSD** [comment]
Yes, this would be fantastic! Like @ryanjamurphy and other people, I use Obsidian to manage writing projects—in my case, news articles and marketing copywriting. Navigating by properties would be extremely helpful!

**Comment by tadashi-aikawa** [comment]
Hi, @ryanjamurphy, @MitchWagnerSD  :)

There is already a similar issue https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/issues/39, but it would be better that this issue is mainstream.

**Comment by craziedde** [comment]
@tadashi-aikawa 

> ..If you want to search for items where the parent property partially matches "antigra," please use `@parent:antigra` instead of `@parent: antigra`.

Sorry for the confusion. I want to search for @property-name: <my-property-name> ONLY, avoiding all other results
Maybe that could be a configurable option?

**Comment by tadashi-aikawa** [comment]
@craziedde 
Thanks for clarifying! Could you please open a new issue with a few concrete examples of your use case? Based on your description, this seems more like a feature request than a bug in AQS, so I'd like to track and discuss it separately.
